### ISSUE TITLE
Please review the Kabanero Pipelines Troubleshooting guide

### ISSUE THREAD

**Issue by aadeshpa** [kabanero-io/docs#116]
Below is the link of the Troubleshooting guide: 
https://github.com/kabanero-io/kabanero-pipelines/blob/83d770f3ff5b4875278dad61b6af33b746386d2a/Troubleshooting.md

Please review this troubleshooting guide and suggest/make changes to improve the readability.   Also if possible and if it makes sense please reformat so it is consistent with other Kabanero troubleshooting guides

**Comment by aadeshpa** [comment]
This is an issue in reference to the issue : https://github.com/kabanero-io/kabanero-pipelines/issues/54

**Comment by nastacio** [comment]
In general, if we know what to tell the user to go run, we should run it ourselves and explain to the user what is wrong.

The idea of written instructions is more useful to cases where the information being collected requires higher-level abstractions and (imprecise) judgement of how to proceed.

For instance, for something like "Check whether your pipelinerun failed due to unbound PersistentVolumeClaims"


we could multiply the inherent goodness of the advice with:

```
oc get pipelineruns -n kabanero  | grep -vi Succeeded | tr -s " " | cut -d " " -f 1
```

to get the name of all failed runs, then use something like 
```
oc get pipelinerun <name_of_failed_run> -n kabanero -template={{.path.to.field}}
```
to get the explanation and so on.

I do realize "so on" may mean hours worth of tearing through k8s CLI docs.

**Comment by bschrammIBM** [comment]
I don't think this information should be a separate guide. It can be included in the Pipelines Guide. See https://github.com/kabanero-io/docs/issues/56

**Comment by jantley-ibm** [comment]
@aadeshpa @nastacio  could you add the information in this issue to issue 56 and close this one?

**Comment by aadeshpa** [comment]
Thank you @nastacio for the feedback, we will try to improve the troubleshooting documentation going forward.

**Comment by nastacio** [comment]
> @aadeshpa @nastacio could you add the information in this issue to issue 56 and close this one?

I don't have additional information for issue #56, I was offering a comment about the approach towards troubleshooting in general.

**Comment by marikaj123** [comment]
@ aadeshpa, can you please help based on the above comment.
